alkira/resource_alkira_connector_gcp_vpc.go

```diff
@@ -55,6 +55,26 @@ func resourceAlkiraConnectorGcpVpc() *schema.Resource {
 				Type:        schema.TypeString,
 				Optional:    true,
 			},
+			"vpc_subnet": {
+				Description: "The list of subnets of the target GCP VPC for routing purpose." +
```

Comment:
For multi lines break, you may want to add an extra space at last like `...purpose. "`. Otherwise, the space won't show up once generating docs.